src/bits_8id/configs/devices.yml

```diff
@@ -0,0 +1,76 @@
+# 8id.devices.AD_Eiger:
+#   - name: eiger4M
+#     prefix: "8id:Eiger4M:"
+#     labels: ["detector", "area_detector"]
+
+apstools.devices.area_detector_factory.ad_creator:
+
+- name: eiger4M
+  prefix: "8idEiger4M:"
+  # IOC host: beryl
+  # IOC top: /net/s8iddserv/xorApps/epics/synApps_6_2_1/ioc/8idEiger4m/iocBoot/ioc8idEiger4m/softioc/8idEiger4m.pl
+  labels:
+  - area_detector
+  - detector
+  - detectors
+  plugins:
+  - badpix1 # apstools 1.7.3+
+  - cam:
+      class: 8id.devices.ad_common.EigerDetectorCam_V34
+  - image
+  - hdf1:
+      class: apstools.devices.area_detector_support.AD_EpicsFileNameHDF5Plugin
+      # TODO #41 switch to the Voyager-based path structure: "/gdata/..."
+      read_path_template: /home/8ididata/2023-2/pvaccess_test/
+      # same filesystem for IOC & Bluesky
+      write_path_template: /home/8ididata/2023-2/pvaccess_test/
+  - overlay1
+  - pva
+  - roi1
+  - stats1
+
+8id.devices.AD_Lambda:
```

Comment:
`8id` is not a valid Python package name.  What is the instrument's package name?  This might change as we add support for the E station.

Reply:
I was wondering the same thing but it seems like this is resolved now.